### PR TITLE
Inst UI 4066 investigation integration of vitest for unit testing

### PR DESCRIPTION
**Pull request by git-nandor**

### CODE REVIEW

**Comment by balzss** [comment]
looks great, I think we should go forward with this

File: packages/ui-popover/src/Popover/__new-tests__/Popover.test.tsx [hunk 1]
```diff
@@ -70,7 +71,7 @@ const PopoverContainer = ({
 
 describe('<Popover />', () => {
   afterEach(() => {
-    jest.resetAllMocks()
+    //jest.resetAllMocks()
```
Comment (joyenjoyer) [inline packages/ui-popover/src/Popover/__new-tests__/Popover.test.tsx:74]: Consider removing this instead of commenting.

**Comment by balzss** [comment]
I guess `jest` as a dependency could also be removed in this pr

**Review by balzss — CHANGES_REQUESTED** [review]
seems like vitest is running in watch mode by default (but not in CI), would be nice to disabled that

**Comment by balzss** [comment]
also package-lock is not in synch with package.json, please update that

**Review by matyasf — CHANGES_REQUESTED** [review]
dev-commands.md needs to be around the "npm run test" part, it still says that the tests are running in Chrome. otherwise good job!

**Review by balzss — APPROVED** [review]
great job!

**Review by matyasf — APPROVED** [review]

**Pull request merged**